Lib/test/test_structseq.py

```diff
@@ -108,7 +156,23 @@ def __len__(self):
 
     def test_reduce(self):
         t = time.gmtime()
-        x = t.__reduce__()
+        cls, (tup, dct) = t.__reduce__()
```

Comment:
Usually we do not have tests for `__reduce__()`. Instead we have tests for pickling and copying. I opened a separate issue for this: #110267.

Reply:
I reverted and moved these changes to another PR:

- #110272